### PR TITLE
win, util: fix the incorrect size of the buffer

### PR DESCRIPTION
**Pull request by uudiin**
The buffer size is the count of characters, not sizeof(buffer).

### CODE REVIEW

File: src/win/util.c [hunk 1]
```diff
@@ -1388,7 +1388,7 @@ int uv__getpwuid_r(uv_passwd_t* pwd) {
   if (OpenProcessToken(GetCurrentProcess(), TOKEN_READ, &token) == 0)
     return uv_translate_sys_error(GetLastError());
 
-  bufsize = sizeof(path);
+  bufsize = sizeof(path) / sizeof(wchar_t);
```
Comment (bnoordhuis) [inline src/win/util.c]: It's a minor thing but it would be better to use `ARRAY_SIZE(path)` here and below.

**Comment by nrstmss** [comment]
Dheh

**Comment by cjihrig** [comment]
Ping @uudiin

**Comment by cjihrig** [comment]
CI: https://ci.nodejs.org/view/libuv/job/libuv-test-commit/450/

**Comment by cjihrig** [comment]
Landed in https://github.com/libuv/libuv/commit/832ab902cf63dd42d6c75b60196dfb83d553e126. Thanks!

**Pull request closed without merging**